test/files/main.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
usually call this file files.go, not main.go (entrypoint for package main)

Reply:
done